docs/getting_started/index.md

```diff
@@ -0,0 +1,17 @@
+# Getting Started
+
+The following docs are a brief introduction to SDK development.
+
+```{toctree}
+:glob:
+:maxdepth: 1
+:hidden:
+
+```
```

Comment:
It looks like this renders as well, is that intentional?

Reply:
I'm not sure what the deal is with these things. i'd like to see it render on the site since the preview shows these on the other index files as well. Will address as a followup if required.